src/index.js

```diff
@@ -42,11 +45,25 @@ const remountUI = function() {
       hideHero={hideHero}
       showAdmin={showAdmin}
       favoriteHubsResult={favoriteHubsResult}
-      report={qs.has("report")}
       installEvent={installEvent}
     />
   );
-  ReactDOM.render(root, document.getElementById("home-root"));
+
+  function returnToRoot() {
+    location.href = "/#/";
+  }
+
+  const router = (
+    <HashRouter>
+      <>
+        {root}
+        <Route path="/joinus" render={() => <JoinUsDialog onClose={returnToRoot} />} />
```

Comment:
should on close just perform the back button?

Reply:
It would be a bit surprising for users who happen to arrive to these dialogs from another to click on a close button and have them go back.